subjects/vowels-index/README.md

```diff
@@ -0,0 +1,41 @@
+## vowels-index
+
+### Instructions
+
+Write a function that takes one argument of type string and returns an array of integers containing the index of the vowels in the string.
+- vowels: [a, e, i, o, u]
+
+### Expected function
+
+```go 
+func VowelsIndex(str string) []int {
+}
+```
+### Usage
+
+Here is a possible program to test your function
+
+```go
+package main
+
+import "fmt"
+
+func main() {
```

Comment:
This program doesn't return the array, you should change this so that the output is the array you ask above...
```
func main() {
	fmt.Println(VowelsIndex("hello Iyan"))
}
```